village/src/components/Smurf.js

```diff
@@ -3,9 +3,18 @@ import React from 'react';
 const Smurf = props => {
   return (
     <div className="Smurf">
-      <h3>{props.name}</h3>
-      <strong>{props.height} tall</strong>
-      <p>{props.age} smurf years old</p>
+      <div className="card bg-light mb-3" >
+        <div className="card-header">{props.name}</div>
+        <div className="card-body">
+          <p className="card-title">{props.id} tall</p>
+          <p className="card-text">{props.height} smurf years old</p>
+          <p className="card-text">{props.age} smurf years old</p>
+        </div>
+        <div class="btn-group" role="group" aria-label="Basic example">
```

Comment:
Remember that `class` is a reserved word, you have to use `className` instead :mag: